src/services/asset-movement/common.ts

```diff
@@ -330,6 +330,12 @@ export function getKeetaAssetMovementAnchorInitiateTransferRequestSigningData(in
 	}));
 }
 
+export type AssetFeeLineItemType = 'RAIL' | 'NETWORK' | 'PROVIDER' | 'VALUE_VARIABLE' | 'OTHER';
+export type AssetFeeBreakdown = {
+	lineItems: { value: string; purpose: AssetFeeLineItemType; }[];
```

Comment:
Can we have `tokenPublicKey` too? And why `value` is string instead of bigint?

```suggestion
	lineItems: { value: bigint; purpose: AssetFeeLineItemType; tokenPublicKey: string }[];
```